### ISSUE TITLE
Cobbler Reposync Unrecognized Argument: --download_path

### ISSUE THREAD

**Issue by zuluetar** [cobbler/cobbler#1836]
Host OS/version: Fedora 26 Server 4.12.5
Cobbler version: cobbler-2.8.1

Description of problem:
Cobbler reposync is using yum's default --download_path option instead of dnf or the -p option

Steps to Reproduce:
1. Create a cobbler repo profile. Sample profile:

Name                           : f26_Everything
Apt Components (apt only)      : 
Apt Dist Names (apt only)      : 
Arch                           : x86_64
Breed                          : yum
Comment                        : 
Createrepo Flags               : <<inherit>>
Environment Variables          : {}
Keep Updated                   : False
Mirror                         : https://mirror.aarnet.edu.au/pub/fedora/linux/releases/26/Everything/x86_64/os/
Mirror locally                 : True
Owners                         : ['admin']
Priority                       : 1
External proxy URL             :
RPM List                       : []
Yum Options                    : {}

2. Run cobbler reposync ex. cobbler reposync --only="f26_Everything"

Actual results:
````
task started: 2017-08-21_145453_reposync
task started (id=Reposync, time=Mon Aug 21 14:54:53 2017)
hello, reposync
run, reposync, run!
creating: /var/www/cobbler/repo_mirror/f26_Everything/config.repo
creating: /var/www/cobbler/repo_mirror/f26_Everything/.origin/f26_Everything.repo
running: /usr/bin/dnf reposync --skip-broken --allowerasing --downloadonly --config=/var/www/cobbler/repo_mirror/f26_Everything/.origin/f26_Everything.repo --repoid=f26_Everything --download_path=/var/www/cobbler/repo_mirror -a x86_64
received on stdout: 
received on stderr: usage: dnf reposync [-c [config file]] [-q] [-v] [--version]
                    [--installroot [path]] [--nodocs] [--noplugins]
                    [--enableplugin [plugin]] [--disableplugin [plugin]]
                    [--releasever RELEASEVER] [--setopt SETOPTS]
                    [--skip-broken] [-h] [--allowerasing] [-b] [-C]
                    [-R [minutes]] [-d [debug level]] [--debugsolver]
                    [--showduplicates] [-e ERRORLEVEL] [--obsoletes]
                    [--rpmverbosity [debug level name]] [-y] [--assumeno]
--repo
                    [repo]] [-x [package]] [--disableexcludes [repo]]
                    [--repofrompath [repo,path]] [--noautoremove]
                    [--nogpgcheck] [--color COLOR] [--refresh] [-4] [-6]
                    [--destdir DESTDIR] [--downloadonly] [--bugfix]
                    [--enhancement] [--newpackage] [--security]
                    [--advisory ADVISORY] [--bzs BUGZILLA] [--cves CVES]
                    [--sec-severity {Critical,Important,Moderate,Low}]
                    [--forcearch ARCH] [-p DOWNLOAD_PATH]
dnf reposync: error: unrecognized arguments: --download_path=/var/www/cobbler/repo_mirror -a x86_64
````

Expected results:

Should create a mirror on the /var/www/cobbler/repo_mirror

**Comment by seano3211** [comment]
The issue is present in the latest Fedora 27 and Cobbler 2.8.2. The issue is in the /usr/lib/python2.7/site-packages/cobbler/action_reposync.py file where the --download_path argument needs to be updated (it's not a valid dnf argument in fedora 26/27) and the -a <arch> needs to be removed as it is also unsupported by dnf.

**Comment by kwizart** [comment]
I think this issue can be closed as a duplicate for issue#1980 (yum is deprecated, should use python3-dnf instead). Maybe this can be workaround by installing yum-utils instead of dnf-utils

**Comment by SchoolGuy** [comment]
I will close this as requested because this is a duplicate.
